client/src/legacy/jquery.changetracker.js

```diff
@@ -65,6 +65,9 @@ import debounce from 'lodash.debounce';
 
     var options = $.extend({}, this.defaults, _options);
 
+    // It is mandatory to never allow the search box or markup in the preview to affect change tracking.
+    options.ignoreFieldSelector += ',.search-box *,.cms-navigator *';
```

Comment:
Adding this here instead of to the default selector list because even if you update the list in a custom section, these should still always be excluded from tracking.